corehq/apps/locations/models.py

```diff
@@ -374,6 +398,15 @@ def by_location_id(cls, location_id):
         except cls.DoesNotExist:
             return None
 
+    def linked_supply_point(self):
+        from corehq.apps.commtrack.models import SupplyPointCase
+        if not self.supply_point_id:
+            return None
+        try:
+            return SupplyPointCase.get(self.supply_point_id)
+        except:
```

Comment:
Incidentally, it looks like `SupplyPointCase.get()` catches `ResourceNotFound` and raises `CaseNotFound`, so that's probably what we want to use.